autoware_reference_system/test/generate_traces.py

```diff
@@ -73,7 +73,24 @@ def generate_test_description():
         ] + DEFAULT_CONTEXT,
     )
 
-    launch_description.add_action(trace_action)
+    trace_memalloc_action = Trace(
+        session_name='profile_' + test_exe_name + '_' + str(RUNTIME) + 's',
```

Comment:
> also @christophebedard are you allowed to add multiple `Trace` actions to one single `LaunchDescription`?

but you shouldn't use the same session name. Sorry I just noticed.

And since the two tracing sessions (resulting from two `Trace` actions) have overlapping events, you'll unnecessarily get duplicate data.

Reply:
@christophebedard can they all be added to just one trace then? Or is that not possible?

Reply:
> @christophebedard can they all be added to just one trace then? Or is that not possible?

Yes you can just merge the events list into one `Trace` action